### ISSUE TITLE
[EN][Spell] Cannibalize

### ISSUE THREAD

**Issue by Fearyourself** [Atlantiss/BugTracker#8432]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: http://cata.openwow.com/spell=20577

When activated, regenerates 7% of total health and mana every 2 sec for 10 sec. Only works on Humanoid or Undead corpses within a1 yds. Any movement, action, or damage taken while Cannibalizing will cancel the effect.

**Current behaviour**: Cannibalize can't be used when the enemy player released spirit.

**Expected behaviour**: Should be able to use it even though someone releases spirit.

**Steps to reproduce the problem**:

1.  Murder someone
2.  Have them release spirit
3.  Use Cannibalize

**Revision**: It works fine except for the fact that you can't use it on players who released which makes it more or less useless in pvp since everyone will release immediately.

**Comment by Zhalox123** [comment]
Yep confirmed even when enemy releases corpse you are not allowed to eat it and u should.Even friendly target can be cannibalazed if he is dead.

**Comment by Piklo** [comment]
you can see current server revision by typing ".s in" in chat
example:
![revision](https://cloud.githubusercontent.com/assets/16057987/19606271/789e530e-97c1-11e6-8047-338129a3a22a.PNG)
